### ISSUE TITLE
The whole codebase proprio_attention_mask is None

### ISSUE THREAD

**Issue by lucasjinreal** [DravenALG/VLANeXt#1]
Hi, looks like the whole codebase proprio_attention_mask is None, does vlm condictioned propri really works? I couldn't understand. Everywhere calls the proprio_attention_mask is just None

**Comment by DravenALG** [comment]
Hi, if you use qwen as the backbone, you can see in the function _get_vlm_condition_qwen in VLANeXt.py, we use "proprio_mask" as the mask, which is not None (line 385 in VLANeXt.py). And then we concat it into the whole attention_mask.

**Comment by DravenALG** [comment]
The whole process is like this (taking libero training as an example):
1. We will prepare proprio_np (line 142 in libero_act.py) in the dataset class, whether we use it or not.
2. In training, we will have a DataCollatorForVLANeXt class to handle the proprio_np, we will prepare proprio_np if self.use_proprio_input_vlm=true (line 273 in train.py)
3. Then, within the model, we will prepare the proprio_mask for inputting to vlm (line 385 in VLANeXt.py). **Because all the positions are valid, we just create a full 1 mask (line 385 in VLANeXt.py) for it.**
